### PR TITLE
more drink options

### PR DESCRIPTION
**Pull request by platinummonkey**

### CODE REVIEW

File: src/commands/drink.rs [hunk 1]
```diff
@@ -15,6 +15,12 @@ fn drink(ctx: &mut Context, msg: &Message) -> CommandResult {
         "Sparking Water",
         "Seltzer Water",
         "Tap Water",
+        "Gravy",
+        "Rum",
+        "Margarita",
+        "Moscow Mule",
```
Comment (mokomull) [inline src/commands/drink.rs]: these do not fit the rest of the theme

**Review by mokomull — CHANGES_REQUESTED** [review]

File: src/commands/drink.rs (hunk 1 above)
Comment (tmcarr) [inline src/commands/drink.rs]: These dont look like water....

File: src/commands/drink.rs [hunk 2]
```diff
@@ -15,6 +15,12 @@ fn drink(ctx: &mut Context, msg: &Message) -> CommandResult {
...
+        "Prairie Oyster",
+        "Pickle Juice",
```
Comment (tmcarr) [inline src/commands/drink.rs]: Holy hell these ARE NOT WATER.

File: src/commands/drink.rs (hunk 1 above)
Comment (platinummonkey) [reply src/commands/drink.rs]: oh, should I have added Taco?

File: src/commands/drink.rs [hunk 3]
```diff
@@ -15,6 +15,8 @@ fn drink(ctx: &mut Context, msg: &Message) -> CommandResult {
         "Sparking Water",
         "Seltzer Water",
         "Tap Water",
+        "Gravy",
```
Comment (tmcarr) [inline src/commands/drink.rs]: Take this out and I'll merge. :P

**Review by tmcarr — CHANGES_REQUESTED** [review]

**Pull request merged**